src/Servers/Kestrel/Core/src/Internal/Http2/Http2StreamOfT.cs

```diff
@@ -12,10 +13,11 @@ internal sealed class Http2Stream<TContext> : Http2Stream, IHostContextContainer
     {
         private readonly IHttpApplication<TContext> _application;
 
-        public Http2Stream(IHttpApplication<TContext> application, Http2StreamContext context) 
+        public Http2Stream(IHttpApplication<TContext> application, Http2StreamContext context, ExecutionContext initialExecutionContext) 
```

Comment:
Curious why execution context is not on Http2StreamContext. Should it be?

Reply:
I like that